test/ItemDetails.test.js

```diff
@@ -0,0 +1,296 @@
+import React from 'react';
+import { shallow, mount, reader } from 'enzyme';
+import MarkdownContainer from '../client/src/CostComponents/MarkdownContainer.jsx';
+import CostContainer from '../client/src/CostComponents/CostContainer.jsx';
+import StyleFlagsContainer from '../client/src/DetailsComponents/StyleFlagsContainer.jsx';
+import MaterialsContainer from '../client/src/DetailsComponents/MaterialsContainer.jsx';
+import DescriptionContainer from '../client/src/DetailsComponents/DescriptionContainer.jsx';
+import DetailsContainer from '../client/src/DetailsComponents/DetailsContainer.jsx';
+import ChoicesContainer from '../client/src/OptionComponents/ChoicesContainer.jsx';
+import PersonalizeContainer from '../client/src/OptionComponents/PersonalizeContainer.jsx';
+import QuantityContainer from '../client/src/OptionComponents/QuantityContainer.jsx';
+import SellingFlagsContainer from '../client/src/OptionComponents/SellingFlagsContainer.jsx';
+import OptionsContainer from '../client/src/OptionComponents/OptionsContainer.jsx';
+import PoliciesContainer from '../client/src//PoliciesComponents/PoliciesContainer.jsx';
+import ShippingContainer from '../client/src/ShippingComponents/ShippingContainer.jsx'
+import TimeframeContainer from '../client/src/ShippingComponents/TimeframeContainer.jsx'
+import ShipSourceContainer from '../client/src/ShippingComponents/ShipSourceContainer.jsx'
+import ItemDetails from '../client/src/ItemDetails.jsx';
+
+describe('basic test of testing with jest', () => {
```

Comment:
I see you created sanity test for jest. This is basically testing to see if jest is working as expected. 

I  understand if your using them as your baseline tests, but you don't really need them.